core-bundle/src/Resources/config/listener.yml

```diff
@@ -273,3 +276,8 @@ services:
             - '@event_dispatcher'
         tags:
             - kernel.event_listener
+
+    Contao\CoreBundle\EventListener\DataContainer\CustomTemplateOptionsCallback:
+        arguments:
+            - '@contao.framework'
+            - '@request_stack'
```

Comment:
Should this add a tag like `{ name: kernel.reset, method: reset }`?

Reply:
Yep, done.